website/content/docs/platform/k8s/helm/configuration.mdx

```diff
@@ -124,7 +124,33 @@ and consider if they're appropriate for your deployment.
         cpu: '250m'
     ```
 
-  - `namespaceSelector` (`dictionary: {}`) - The selector used by the admission webhook controller to limit what namespaces where injection can happen. If set to null, all non-system namespaces are eligible for injection.
+  - `webhook` - Values that control the Mutating Webhook Configuration.
+
+    - `failurePolicy` (`string: "Ignore"`) - Configures failurePolicy of the webhook. To block pod creation while the webhook is unavailable, set the policy to `Fail`. See [Failure Policy](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#failure-policy).
+
+    - `matchPolicy` (`string: "Exact"`) - Specifies the approach to accepting changes based on the rules of the MutatingWebhookConfiguration. See [Match Policy](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-matchpolicy).
+
+    - `timeoutSeconds` (`int: 30`) - Specifies the number of seconds before the webhook request will be ignored or fails. If it is ignored or fails depends on the `failurePolicy`. See [timeouts](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts).
+
+    - `namespaceSelector` (`object: {}`) - The selector used by the admission webhook controller to limit what namespaces where injection can happen. If unset, all non-system namespaces are eligible for injection. See [Matching requests: namespace selector](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-namespaceselector).
+
+      ```yaml
+      namespaceSelector:
+        matchLabels:
+          sidecar-injector: enabled
+      ```
+
+    - `objectSelector` (`object: {}`) - The selector used by the admission webhook controller to limit what objects can be effected by mutation.. See [Matching requests: object selector](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-objectselector).
+
+      ```yaml
+      objectSelector:
+        matchLabels:
+          sidecar-injector: enabled
+      ```
+
+    - `annotations` (`string or object: {}`) - Defines additional annotations to attach to the webhook. This can either be YAML or a YAML-formatted multi-line templated string.
+
+  - `namespaceSelector` (`dictionary: {}`) - Deprecated: please use 'webhook.namespaceSelector' instead.
```

Comment:
Could link to the new namespaceSelector line here if we wanted (not sure if the syntax is completely correct):

```suggestion
  - `namespaceSelector` (`dictionary: {}`) - Deprecated: please use [`webhook.namespaceSelector`](/docs/platform/k8s/helm/configuration#namespaceselector) instead.
```